src/js/CliAutoComplete.js

```diff
@@ -291,7 +291,7 @@ CliAutoComplete._initTextcomplete = function() {
          */
         if (!savedMouseoverItemHandler) {
             // save the original 'mouseover' handeler
-            savedMouseoverItemHandler = $._data($('.textcomplete-dropdown')[0], 'events').mouseover[0].handler;
+            savedMouseoverItemHandler = $._data($('.textcomplete-dropdown')[0], 'events').mouseover[0]?.handler;
```

Comment:
Why was this added?

Reply:
Fixed

Reply:
O no wait this was due to a intermittent bug where mouseover is sometimes not defined at first refererence. (https://github.com/betaflight/betaflight-configurator/issues/2299)